kolibri/content/api.py

```diff
@@ -281,6 +283,43 @@ def all_content(self, request, **kwargs):
         return Response(serializer.data)
 
 
+class ContentNodeGranularViewset(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+    serializer_class = serializers.ContentNodeGranularSerializer
+
+    def get_queryset(self, available=False):
+        if available:
```

Comment:
So this would become:
```
if available is not None:
    queryset = models.ContentNode.objects.filter(available=available)
```
I know you're not necessarily wanting to filter by `available=False` but it just feels confusing otherwise.